kafka-connector-test/pom.xml

```diff
@@ -1,31 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.axonivy.connector.kafka</groupId>
   <artifactId>kafka-connector-test</artifactId>
-  <version>10.0.9-SNAPSHOT</version>
+  <version>11.2.0-SNAPSHOT</version>
   <packaging>iar</packaging>
   <dependencies>
     <dependency>
       <groupId>com.axonivy.connector.kafka</groupId>
       <artifactId>kafka-connector</artifactId>
-      <version>10.0.9-SNAPSHOT</version>
+      <version>${project.version}</version>
```

Comment:
:liberia:

Reply:
What does LR mean?

Reply:
was just posting a victorious flag; cheering your efforts to simplify the dependencies here :)
sorry, will try to use less emojis. Always forget that windows users can't see it

Reply:
I can see it using firefox but some are missing with Edge :-(